### DISCUSSION TITLE
Start search at default branch within a repository

### DISCUSSION THREAD

**Discussion by gravyboat** [community/community#9019] [Other Feature Feedback, Questions, & Ideas] [answered]
One of my primary use cases when doing searches (probably 99% of my searches) is to see code that is currently merged to the default branch in a repository whether that be main, develop, or something else. Currently when searching you are presented with results from multiple branches (old branches that are still around and contain inaccurate code). This issue exists within the current search as well as Code Search.

There's some discussion [here](https://github.com/github/feedback/discussions/8564) and @p0's [comment](https://github.com/github/feedback/discussions/8564#discussioncomment-1814115) is specifically what I am mentioning as the default behavior which would greatly improve my Code Search experience.

**Comment by gorzell** [comment, marked as answer]
Do you have an example of a query where you are getting results in a non-default branch?  I don't think that this should be happening in the new cs.github.com experience at this point.  If were to add it, it would only be likely to show up in cases where there were no on default branches.

**Reply by gravyboat** [reply]
I tried to duplicate it this morning and couldn't do so on a public repo or a private repo. Was this fixed recently? I had this issue when I originally created this thread and referenced the other issue. I'll mark this as answered.
